src/Integration.Vsix/Resources/Strings.resx

```diff
@@ -129,16 +129,6 @@
     <value>Cannot find tool window with the specified guid: {0}</value>
     <comment>Exception message, {0} guid</comment>
   </data>
-  <data name="Daemon_Downloaded" xml:space="preserve">
```

Comment:
There are still a number of strings prefixed with `Daemon_` in this resource file, but they are all still in use. In so far as possible I'll keep deletions and renames in separate commits to make the changes easier to review.